### ISSUE TITLE
Flashing, crashing, having a rave party

### ISSUE THREAD

**Issue by Elyssen-1** [murrty/YChanEx#10]
Worked fine yesterday, now it just starts flashing everything and crashes seconds later.

Gif about the issue:
https://i.imgur.com/NabzHCp.gif

Error it gives me:
 https://pastebin.com/raw/GKwDNHgV

I've also never got the "Userscript" to install, the button does nothing.
As I've understood it's supposed to give me a button on the site to download using :ychanex ?

**Comment by murrty** [comment]
I resolved, rather not elegantly, the flashing problem. It's because the window thread was waiting for each thread to load separately, causing it the windows to flash.

I can't really resolve the other problem, I don't really know why it's happening, and I cannot reproduce it either. I need to either run into a thread that can reproduce this issue, or if you still have it (judging by the date created, probably not) happening, I can look into it on my end.

Yes, I did not add an event for that button, so it does do anything. For good reason, I don't really have a userscript ready for general use, so I did not include it, as it needs to fit in with the *chan (in most cases). I need to do more work on the backend to make it play nice and work.
